Software/src/SettingsWindow.ui

```diff
@@ -213,7 +213,7 @@
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
-       <number>5</number>
+       <number>1</number>
```

Comment:
Is this diff intentional?

Reply:
No, same with `.travis` file, I'm not sure how this got there, I didn't commit/push these on that branch, I must've messed up my merges somewhere, I'll try to avoid to this. You can skip them if that's possible.